webpack.config.js

```diff
@@ -49,7 +49,7 @@ module.exports = {
   module: {
     rules: [
       {
-        test: /\.js?$/,
+        test: /\.jsx?$/,
```

Comment:
LOL, looks like the previous regex would have matched both `foo.j` and `foo.js` - was that intentional?
Whereas now it'll only match `foo.js` or `foo.jsx`